shared/src/TwinCollection.cs

```diff
@@ -285,14 +296,33 @@ private void TryClearMetadata(string propertyName)
         }
 
         /// <summary>
-        /// Clear metadata out of the collection
+        /// Clears metadata out of the properties collection.
```

Comment:
By properties collection do you mean twin collection?

Reply:
I do, and that is more clear